Dockerfile.alpine

```diff
@@ -0,0 +1,44 @@
+###
+# Build Stage.
+FROM redis:5-alpine as builder
```

Comment:
One change request - can we use `redis:6-alpine` as the base image? We are generally migrating in that direction, so we could be proactive by adopting it here!

Reply:
That's fine by me. Let me push one other small change, just a moment.